runtime/Java/src/org/antlr/v4/runtime/misc/Utils.java

```diff
@@ -61,10 +61,10 @@ public  static <T> void removeAllElements(Collection<T> data, T value) {
 	public static String escapeWhitespace(String s, boolean escapeSpaces) {
 		StringBuilder buf = new StringBuilder();
 		for (char c : s.toCharArray()) {
-			if ( c==' ' && escapeSpaces ) buf.append('\u00B7');
-			else if ( c=='\t' ) buf.append("\\t");
-			else if ( c=='\n' ) buf.append("\\n");
-			else if ( c=='\r' ) buf.append("\\r");
+			if (c == ' ' && escapeSpaces) buf.append('\u00B7');
+			else if (c == '\t') buf.append("\\t");
+			else if (c == '\n') buf.append("\\n");
+			else if (c == '\r') buf.append("\\r");
```

Comment:
Please don't mix whitespace and semantics changes (ideally, don't change whitespaces without need).